### PR TITLE
UX: Hide campaign button on goal completion

### PR DESCRIPTION
**Pull request by justindirose**
Simple change -- upon goal completion, hide the "Support" button on the campaign banner.

### CODE REVIEW

File: assets/javascripts/discourse/templates/components/campaign-banner.hbs [hunk 1]
```diff
@@ -13,22 +13,21 @@
...
       background-repeat: no-repeat;' ''}}">
       {{#if isGoalMet}}
-         <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.success_title'}}</h2>
+        <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.success_title'}}</h2>
         <p class="campaign-banner-info-description">{{i18n 'discourse_subscriptions.campaign.success_body'}}</p>
       {{else}}
         <h2 class="campaign-banner-info-header">{{i18n 'discourse_subscriptions.campaign.title'}}</h2>
         <p class="campaign-banner-info-description">{{i18n 'discourse_subscriptions.campaign.body'}}</p>
+        {{#if product}}
+          {{#link-to "s.show" product disabled=product.subscribed class="btn btn-primary campaign-banner-info-button"}}
```
Comment (eviltrout) [inline assets/javascripts/discourse/templates/components/campaign-banner.hbs:22]: I believe I've said this before but I think it's technical debt that the route is called `s`. I'm fine with the path being `/s` but the route should be called something more descriptive like `subscriptions` and we should fix this in the future.

**Review by eviltrout — APPROVED** [review]

File: assets/javascripts/discourse/templates/components/campaign-banner.hbs (hunk 1 above)
Comment (justindirose) [reply assets/javascripts/discourse/templates/components/campaign-banner.hbs:22]: You got it -- I'll add it to the list to refactor in the future.

File: assets/stylesheets/common/campaign.scss [hunk 2]
```diff
@@ -244,6 +245,20 @@ body.archetype-regular {
   }
 }
 
+html:not(.mobile-view) {
```
Comment (jjaffeux) [inline assets/stylesheets/common/campaign.scss:248]: Kinda nitpicking here but why this instead of `html.desktop-view` ?

Comment (justindirose) [reply assets/stylesheets/common/campaign.scss:248]: Just following existing pattern that @jordanvidrine had used.

Comment (jordanvidrine) [reply assets/stylesheets/common/campaign.scss:248]: That's my fault! `html.desktop-view` is better.

**Pull request merged**